app/routes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
+
+// Route to UPDATE the config
+router.post('/configs', function (req, res) {
+	var configData = config.update(req);
```

Comment:
req and res are objs. they have paramaters. you need to create or map a config method onto them rather than calling the config method directly. its alot cleaner and faster this way.